### PR TITLE
fix(api): fix database seeding (BOAS-118)

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/prisma/seed.js [hunk 1]
```diff
@@ -719,6 +620,7 @@ async function main() {
 				{ shorthand: 'HAS', type: appealTypes.HAS }
 			]
 		});
+		// await prisma.appeal.create({ data: appealsData[3] });
```
Comment (francescobrenci-AR) [inline apps/api/prisma/seed.js:623]: is this comment needed?

**Review by francescobrenci-AR — DISMISSED** [review]

File: apps/api/prisma/seed.js (hunk 1 above)
Comment (mayya-bond-kc) [reply apps/api/prisma/seed.js]: ah no, removed it now

**Review by DavidJWhiteDev — APPROVED** [review]
tested and fully working

**Pull request merged**